### DISCUSSION TITLE
Running the Project

### DISCUSSION THREAD

**Discussion by Deebling5** [shubham1710/Fenice-Network#6] [General]
Hey Shubham,

how do I run this projects. I am facing challenges.
I went through your blog on Medium but even then its always giving errors. New errors each time I edit and do something.

Can you write a few commands on how can one run this project in its raw state. 
Thanks in advance

**Comment by shubham1710** [comment]
Actually, I have implemented quite a few things like Social login in which you need to set up with those providers. There are various tutorials on these topics. There would be problems to connect to the different databases for development and production. 

Please try to get the answers on Stackoverflow since I too had to deal with errors for hours searching and debugging. You will actually learn a lot while doing it. 

Sorry, I cannot help you out in this now since I am little busy with other things.

**Reply by ymadhubabu** [reply]
Hi subham 
I have setup the virtual environment  and installed all dependencies from piplock file. But i could not manage to run the application on django development server. It is reporting following errors . Could you please help me in this regard if time permits

<img width="899" alt="Capture" src="https://user-images.githubusercontent.com/22412676/117383533-9a626d00-aefe-11eb-8076-4188141be299.PNG">

[errors.docx](https://github.com/shubham1710/Fenice-Network/files/6438338/errors.docx)

**Reply by shubham1710** [reply]
This is configuration issue. You have not defined the Secret Key in your environment files so it is showing secret key cannot be empty. You have to use env variables to store those sensitive info.

**Comment by ymadhubabu** [comment]
> Hey Shubham,
> 
> how do I run this projects. I am facing challenges.
> I went through your blog on Medium but even then its always giving errors. New errors each time I edit and do something.
> 
> Can you write a few commands on how can one run this project in its raw state.
> Thanks in advance

Hi did u succeed running this project on django development server...please hep me run

**Reply by Deebling5** [reply]
Hi. So I can see you have a issues with Secret key. Just remove the part where its picking up from environment variables and manually give it a key

SECRET_KEY = '1c4^_7s#e@3e+#800n$ej=l@m*p(rlme9*75272i!c10+l3f&^'
DEBUG = True
ALLOWED_HOSTS = [*]

And apart from that, remove sql db and everything from migrations folder except init file. Then run make migrations and migrate command. New Database will be created and I hope you'll be able to run the code.

**Reply by shubham1710** [reply]
Thanks a lot for your explanations. Hope you were able to run it correctly @ymadhubabu 
Sorry I could not help much since I was busy with my college work.

**Comment by craig-gigged** [comment]
Hi all, 

I am new to using python but have been really enjoying these courses. I am having issues with styling on the JobBoard course. Have i done something wrong or is it as simple as the CSS files are no longer accessible from where they were originally? 

Any help would be greatly appreciated 😊

<img width="1562" alt="Screenshot 2023-12-17 at 12 29 45" src="https://github.com/shubham1710/Fenice-Network/assets/77725398/7a27d2c9-2e7b-42af-a865-d97765f5a0e6">
